addons/project/tests/test_project_stage_multicompany.py

```diff
@@ -0,0 +1,94 @@
+from .test_multicompany import TestMultiCompanyProject
+
+from odoo.exceptions import UserError
+
+
+class TestProjectStagesMulticompany(TestMultiCompanyProject):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestProjectStagesMulticompany, cls).setUpClass()
+
+        Users = cls.env['res.users'].with_context({'no_reset_password': True})
+        cls.user_manager_companies = Users.create({
+            'name': 'Manager Companies',
+            'login': 'manager-all',
+            'email': 'manager@companies.com',
+            'company_id': cls.company_a.id,
+            'company_ids': [(4, cls.company_a.id), (4, cls.company_b.id)],
+            'groups_id':
+                [(6, 0, [
+                    cls.env.ref('base.group_user').id,
+                    cls.env.ref('project.group_project_stages').id,
+                    cls.env.ref('project.group_project_manager').id,
+                ])]
+        })
+        cls.stage_company_a = cls.env['project.project.stage'].create({
+            'name': 'Stage Company A',
+            'company_id': cls.company_a.id,
+        })
+        cls.stage_company_b = cls.env['project.project.stage'].create({
+            'name': 'Stage Company B',
+            'company_id': cls.company_b.id,
+        })
+        cls.stage_company_none = cls.env['project.project.stage'].create({
+            'name': 'Stage Company None',
+        })
```

Comment:
```suggestion
        cls.stage_company_a, cls.stage_company_b, cls.stage_company_none = cls.env['project.project.stage'].create([{
            'name': 'Stage Company A',
            'company_id': cls.company_a.id,
        }, {
            'name': 'Stage Company B',
            'company_id': cls.company_b.id,
        }, {
            'name': 'Stage Company None',
        }])
```

Reply:
Every time I do it :sweat: This is hard to remember lol, thanks :)

Reply:
That's ok, it'll become natural :)